packages/input-option/src/InputOption/InputOption.types.ts

```diff
@@ -26,9 +44,9 @@ export interface BaseInputOptionProps {
   highlighted?: boolean;
 
   /**
-   * Whether the component is selected, regardless of keyboard navigation
+   * Whether the component is checked, regardless of keyboard navigation
    */
-  selected?: boolean;
+  checked?: boolean;
```

Comment:
Didn't we align on `selected` to avoid confusion with checkboxes where `checked` is a native HTML prop?

Reply:
we have docs on this and I'm 99% sure that we aligned on checked

Reply:
`checked`: https://docs.google.com/document/d/1HlzHBlcL5MZG5Lm5m-OIfkI0Dhh4x1XBezLUipV3SW0/edit#heading=h.f9ili2y9gq48